CHANGELOG.md

```diff
@@ -20,11 +20,20 @@ Version <dev> gleans Docker container IDs from cgroups v2-based containers, reco
 
   For compatibility with Ruby 3.4 and to silence compatibility warnings present in Ruby 3.3, declare a dependency on the `base64` gem. The New Relic Ruby agent uses the native Ruby `base64` gem for Base 64 encoding/decoding. The agent is joined by Ruby on Rails ([rails/rails@3e52adf](https://github.com/rails/rails/commit/3e52adf28e90af490f7e3bdc4bcc85618a4e0867)) and others in making this change in preparation for Ruby 3.3/3.4. [PR#2238](https://github.com/newrelic/newrelic-ruby-agent/pull/2238)
 
-- **Fix: Stop sending duplicate log events for Rails 7.1 users**
+-**Feature: Add Roda support for the newrelic_ignore\* family of methods**
+
+  The agent can now selectively disable instrumentation for particular requests within Roda applications. Supported methods include:
+  - `newrelic_ignore`: ignore a given route.
+  - `newrelic_ignore_apdex`: exclude a given route from consideration in overall Apdex calculations.
+  - `newrelic_ignore_enduser`: prevent automatic injection of the page load timing JavaScript when a route is rendered.
+
+  For more information, see [Roda Instrumentation](https://docs.newrelic.com/docs/apm/agents/ruby-agent/instrumented-gems/roda-instrumentation/). [PR#2267](https://github.com/newrelic/newrelic-ruby-agent/pull/2267)
```

Comment:
Do you have a separate issue to track updating these docs to include the ignore methods?

Reply:
I don't yet, but it's next on the list! Happy to hold this merge until we can get a docs PR going.